pcsx2/gui/MainFrame.cpp

```diff
@@ -610,20 +610,20 @@ void MainEmuFrame::ApplyCoreStatus()
 		if( !CoreThread.IsClosing() )
 		{
 			susres->Enable();
-			susres->SetItemLabel(_("Pause"));
+			susres->SetItemLabel(_("&Pause"));
 			susres->SetHelp(_("Safely pauses emulation and preserves the PS2 state."));
 		}
 		else
 		{
 			susres->Enable(vm);
 			if( vm )
 			{
-				susres->SetItemLabel(_("Resume"));
+				susres->SetItemLabel(_("Resu&me"));
 				susres->SetHelp(_("Resumes the suspended emulation state."));
 			}
 			else
 			{
-				susres->SetItemLabel(_("Pause/Resume"));
+				susres->SetItemLabel(_("&Pause/Resume"));
```

Comment:
Technically there isn't a point in adding a shortcut key for this - it'll always be in a disabled state.